.env.example

```
@@ -4,4 +4,9 @@ POSTGRES_USER=agent_board
 AGENT_BOARD_SERVER_PORT=3001
 AGENT_BOARD_WEB_PORT=3000
 AGENT_BOARD_WORKSPACE_ROOT=/var/lib/agent-board/workspaces
+# Compose bind-mount source and its backend-visible destination.
+AGENT_BOARD_REPOSITORY_ROOT=.
+AGENT_BOARD_REPOSITORY_MOUNT_PATH=/repositories
+# Server-side allowlist. Use the backend-visible paths, separated by the OS path-list separator.
+AGENT_BOARD_REPOSITORY_ROOTS=/repositories
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AGENT_BOARD_REPOSITORY_ROOTS key should go before the AGENT_BOARD_SERVER_PORT key

(UnorderedKey)